packages/backend/scripts/20240830_import_devcon_vouchers.js

```diff
@@ -0,0 +1,52 @@
+require("dotenv").config();
+const fs = require("fs");
+const { parse } = require("csv-parse/sync");
+const firebaseAdmin = require("firebase-admin");
+const { encryptData } = require("../utils/encrypt");
+
+const CSV_FILE_PATH = "./devcon_vouchers.csv";
+
+if (process.env.GOOGLE_APPLICATION_CREDENTIALS) {
+  firebaseAdmin.initializeApp({
+    credential: firebaseAdmin.credential.applicationDefault(),
+    storageBucket: "buidlguidl-v3.appspot.com",
+  });
+} else {
+  console.log("Using Firestore emulator");
+  firebaseAdmin.initializeApp({
+    projectId: "buidlguidl-v3",
+    storageBucket: "buidlguidl-v3.appspot.com",
+  });
+}
+
+const db = firebaseAdmin.firestore();
+
+async function importDevconVouchers() {
+  // Read and parse the CSV file
+  const fileContent = fs.readFileSync(CSV_FILE_PATH, "utf-8");
+  const records = parse(fileContent, { columns: true });
+
+  const vouchers = records.map(row => ({
+    voucher: encryptData(row["voucher"]),
+    type: row["type"],
+    builderAddress: "",
+  }));
+
+  console.log(`Parsed ${vouchers.length} vouchers from CSV.`);
+
+  // Import vouchers to the database
+  const batch = db.batch();
+  for (const voucher of vouchers) {
+    const docRef = db.collection("devconVouchers").doc();
+    batch.set(docRef, voucher);
+  }
+
+  try {
+    await batch.commit();
+    console.log("Voucher import completed successfully.");
+  } catch (error) {
+    console.error("Error importing vouchers:", error);
+  }
+}
+
+importDevconVouchers().catch(console.error);
```

Comment:
```js
importDevconVouchers()
  .then(() => {
    process.exit();
  })
  .catch(error => {
    console.error("Error importing vouchers:", error);
    process.exit(1);
  });
```

It seems if you don't do `process.exit()` the process doesn't exit itself, lol I waiting and thinking why its taking so long time.

Reply:
![image](https://github.com/user-attachments/assets/cc9f244b-2880-47bf-bdb5-a519a372b7ef)

I can't seem to reproduce this. It works (instantly) for me and the same for existing scripts. Maybe your node + macos issue? You are running it in the backend folder, right? Like `node scripts/2024......js`

I can add it anyway, since it won't hurt.

Reply:
> Maybe your node + macos issue? You are running it in the backend folder, right? Like node scripts/2024......js

Yeah strange actually might be that case! Because I think I had same problem with some other projects seed script too 


https://github.com/user-attachments/assets/8ff8498a-06ac-4e6a-bbaa-31be547b3429